charts/paperless/values.yaml

```diff
@@ -0,0 +1,75 @@
+# Default values for Paperless.
+
+image:
+  repository: thepaperlessproject/paperless
+  pullPolicy: IfNotPresent
+  tag: latest
+  args: "['gunicorn', '-b', '0.0.0.0:8000']"
+
+strategy:
+  type: Recreate
+
+# See the following files for supported environment variables
+# https://github.com/the-paperless-project/paperless/blob/master/docker-compose.env.example
+# https://github.com/the-paperless-project/paperless/blob/master/paperless.conf.example
+# Be sure to set PAPERLESS_OCR_LANGUAGES as a minimum (See https://github.com/the-paperless-project/paperless/blob/master/docker-compose.yml.example)
+env: {}
+  # TZ: Europe/London
+  # PAPERLESS_OCR_LANGUAGES: eng
+  # PAPERLESS_FORGIVING_OCR: true
+  # PAPERLESS_DISABLE_LOGIN: true
+  # PAPERLESS_INLINE_DOC: true
+  # PAPERLESS_OCR_THREADS:
+  # PAPERLESS_CONVERT_DENSITY:
+
+service:
+  port:
+    port: 8000
+
+persistence:
+  data:
+    enabled: true
+    mountPath: /usr/src/paperless/data
+    accessMode: ReadWriteOnce
+    size: 1Gi
+    skipuninstall: true
```

Comment:
This can be left out by default. The common library will handle this if the chart user provides these keys.